internal/install/application_configuration_yml.go

```diff
@@ -0,0 +1,17 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package install
+
+const (
+	elasticAgentImageName  = "docker.elastic.co/beats/elastic-agent"
+	elasticsearchImageName = "docker.elastic.co/elasticsearch/elasticsearch"
+	kibanaImageName        = "docker.elastic.co/kibana/kibana"
+)
+
+const applicationConfigurationYmlFile = "config.yml"
+
+const applicationConfigurationYml = `stack:
+  image_ref_overrides:
```

Comment:
Maybe put some commented out structure here so it's easy to know how to change it when we need to make overrides? Also maybe include a comment about informing package developers whenever overrides are defined or removed here?